### DISCUSSION TITLE
RabbitMQ cluster name different across nodes

### DISCUSSION THREAD

**Discussion by peterj1974** [rabbitmq/rabbitmq-server#11355] [Questions] [answered]
When setting up a RabbitMQ cluster, the cluster name displayed by the rabbitmqctl cluster_status command varies across nodes. I expected the cluster name would show the node to which other nodes were joined. However, on all three nodes in my cluster, the cluster name corresponds to the individual node names of each respective node.
Is this expected behaviour?

**Comment by lukebakken** [comment, marked as answer]
Yes, you must set the cluster name yourself - https://www.rabbitmq.com/docs/man/rabbitmqctl.8#set_cluster_name
